ibis/backends/datafusion/compiler.py

```diff
@@ -467,6 +468,23 @@ def elementwise_udf(op):
     return udf(*args)
 
 
+@translate.register(ops.ScalarUDF)
+def scalar_udf(op):
+    if (input_type := op.__input_type__) != InputType.PYARROW:
```

Comment:
At some point we may want encode the input types as separate classes to make translation code a bit cleaner.